cylc/flow/task_state.py

```diff
@@ -198,6 +198,17 @@ class RunMode:
     JOBLESS_MODES = frozenset({SKIP, SIMULATION})
     """Modes which completely ignore the standard submission path."""
 
+    MODE_DESCRIPTIONS = {
+        LIVE: "Tasks will run normally.",
+        SIMULATION: (
+            "Simulates job submission with configurable execution time "
+            "(does not submit real jobs)."),
+        DUMMY: "Submits real jobs with empty scripts.",
+        SKIP: (
+            "Skips job submission; sets required outputs (by default) or "
+            "configured outputs.")
+    }
+
```

Comment:
You can merge this with the enum which would keep the source and docs together in one place.

Take a look at how the `StopMode` enumeration does it:

https://github.com/cylc/cylc-flow/blob/4b58a6a7f53f69789466ac88b57832b1433b1679/cylc/flow/workflow_status.py#L66-L138

This is integrated with the GraphQL schema such that the descriptions get suched into the description.